   <style type="text/css">
	 .importantNote{
	  max-width: 1200px;
	  padding: 1em;
	  border-radius:8px;
	  border:1px;
	  border-style: solid;
	  background:#ADD8E6;
	}
	 </style>
  <mat-toolbar>
  <mat-toolbar-row style="background-image: linear-gradient(-133deg, #246A92 0%, #245571 100%)"></mat-toolbar-row>
  <mat-toolbar-row style="background-image: linear-gradient(-133deg, #246A92 0%, #245571 100%)"></mat-toolbar-row>
</mat-toolbar>
<div class="row" style="font:Lato; margin-top:-75px!important; margin-left:20px; margin-right: 20px; align-items: center;" >
<mat-card class="about-page card card-body pageMargins">
  <mat-card-content style="padding-right: 55px;padding-left: 15px;display: block; margin: auto auto; vertical-align: middle;">
  <div class="row"><h3>Data Harmonization</h3></div>
  <div class="row">
    <p>
      The goal of PDC data harmonization process is to transform proteomic data from disparate sources and workflows into derived forms by common data analysis methods and tools. The harmonization process is therefore useful for removing data analysis variables, enabling comparisons across datasets.
    </p>
    <p>
       Harmonization starts with assigning standard identifiers, data integrity checks, adherence to standards (community-accepted vocabulary and nomenclature for clinical attributes, peptides, proteins, protein sequence variants, and modifications) and PDC data model.
    </p>
  </div>
  <div class="row"><h3>Proteomic Data Harmonization</h3>
  </div>
  <div class="row">
    <p>
      The PDC uses submitted raw mass spectrometry data files to produce derived analysis results which can be used to study identification of proteins and post-translational modifications (PTMs).  All processing is done through Common Data Analysis Pipelines (CDAP).  Whenever possible, quantitative results are also extracted from the raw data, enabling downstream analyses of differential expression between samples at the protein- or PTM site-level.
    </p>
  </div>
  <div class="row"><h3>Common Data Types</h3>
  </div>
  <div class="row">
    <p>
      One goal of the PDC is to harmonize a diversity of proteomics data types.  All current data types are mass spectrometry data acquired using data dependent acquisition (DDA), but pipelines are under construction for the analysis of data acquired using data independent acquisition (DIA) or SWATHTM.  DDA deposited in the PDC so far includes label-free, iTRAQ4 and TMT10.  The CDAP is capable of processing data from unenriched, phospho-, and glyco-enriched peptide samples.
    </p>
  </div>
  <div class="row"><h3>Common Data Analysis Pipeline for DDA</h3>
  </div>
  <div class="row">
    <p>
      PDC uses the CDAP developed at NIST and Georgetown University and in use at  the CPTAC Data Coordinating Center
      (published <a href="https://www.ncbi.nlm.nih.gov/pubmed/26860878" target="_blank">here</a>)
      as a starting point for DDA datasets. The pipeline is implemented in the Galaxy Framework that runs on Amazon Cloud. Software programs and parameters are also detailed in <a href="assets/CDAP_ProteinReports_description_20180914.pdf" target="_blank">this</a> document.  In general, the pipeline proceeds in the following order:
    </p>
  </div>
  <div class="row" style="margin: auto auto; vertical-align: middle;">
      <ul>
        <li>
           Spectral conversion to standard formats from proprietary.  This step also extracts peak area values for any reporter ion intensities, if an isobaric labeling strategy has been used (current tool: ReAdW4Mascot2.exe from NIST)
        </li>
        <li>
           Sequence database search (current tool: <a href="https://omics.pnl.gov/software/ms-gf" target="_blank">MSGF+</a> from PNNL)
        </li>
        <li>
           MS1 data processing used for label-free quantitation (current tool: ProMS from NIST)
        </li>
        <li>
           Phosphosite localization (i.e., scoring) if the sample has been enriched (current tool: <a href="http://ms.imp.ac.at/index.php?action=phosphors" target="_blank">PhosphoRS</a> from the Mechtler Lab)
        </li>
        <li>
           Generalized parsimony and “roll-up” of any quantitative information to the gene level (from the spectrum and peptide-levels).  This step also includes normalization and calculation of individual aliquot values to log2ratio values for iTRAQ4 and TMT10 studies. (current tool: PSMLab from the <a href="https://edwardslab.bmcb.georgetown.edu/" target="_blank">Edwards Lab</a> at Georgetown)
        </li>
     </ul>
  </div>
  <div>
    <img src="./assets/css/images/PDC-CDAP-DDA.png" alt="PDC CDAP DDA Image"/>
  </div>
  <div class="row">
    <p>
      The results of the pipeline for a multiplexed labeling study is a matrix in which rows are genes and columns are aliquots (samples).  Values in the cells represent protein (gene) expression for that sample relative to a common reference (typically a pooled sample).
    </p>
  </div>
   <!-- @@@PDC-6981 update data analysis/harmonization -->
  <div class="importantNote">
  <h3>Useful resources:</h3>
	<p>
		An overview of the Common Data Analysis pipeline is outlined <a href="assets/CDAP_Results_Overview_rev_09152014.pdf" target="_blank">here</a> and described in more details in this <a href="https://www.ncbi.nlm.nih.gov/pubmed/26860878" target="_blank">publication</a>.
	</p>
	<p>
		Details of the processed outputs from CDAP can be found <a href="https://pdc.cancer.gov/pdc/faq/Files_Download" target="_blank">here</a>.
	</p>
	<p>
		Detail of the formats of CDPA out puts can be found here : <a href="assets/CPTAC_DCC_mzIdentML_Format_Peptide-Spectrum-Matches_version_1.1.0.pdf" target="_blank">Peptide Spectral Matches</a> and <a href="assets/CDAP_ProteinReports_description_20160503.pdf" target="_blank">Protein Reports</a>.
	</p>
  </div>
  <div class="row"><h3>Common Data Analysis Pipeline for DIA</h3>
  </div>
  <div class="row">
    <p>
	  Our approach for analysis of data independent acquisition is based on work in the <a href="https://bitbucket.org/searleb/encyclopedia/wiki/Home" target="_blank">MacCoss Lab</a> and is currently under development. If available, sample or pool-specific data are used to build a spectral library.  The peptides in the library are then scored assigned match scores for each DIA data file.
    </p>
  </div>
  <div class="row" style="margin: auto auto; vertical-align: middle;">
    <ul>
      <li>
         <!-- @@@PDC-5591-update-site-link -->
         <!-- @@@PDC-8135-update-->
         Spectral conversion to standard formats from proprietary (current tool: <a href="https://proteowizard.sourceforge.io/doc_users.html" target="_blank">msconvert</a>)
      </li>
      <li>
        An individual peptide centric search is performed on each file with <a href="https://bitbucket.org/searleb/encyclopedia/wiki/Home" target="_blank">EncyclopeDIA</a> either using a user supplied spectral library or the pan-human spectral library.
      </li>
      <li>
        An experiment level q-value is calculated using <a href="https://www.ncbi.nlm.nih.gov/pubmed/27572102" target="_blank">Percolator</a> from the results from each individual search, and the results are combined into a single chromatogram library.
      </li>
      <li>
         The peak boundaries and transitions in the chromatogram library are imported into a <a href="https://skyline.ms/project/home/software/Skyline/begin.view" target="_blank">Skyline</a> document to enable quantification and visualization of the results.
      </li>
   </ul>
  </div>
  <div>
    <!--dev-8135-update-DIA-content-->
    <img src="./assets/css/images/PDC-CDAP-DIA-updated.png" alt="PDC CDAP DIA Image"/>
  </div>
  <div class="row"><h3>Availability of Results</h3>
  </div>
  <div class="row">
    <p>
      Following analysis by either CDAP, the results are available as Protein Assembly reports on the PDC Data Portal along with original and processed RAW data and metadata according to approved release schedules.
    </p>
  </div>
  </mat-card-content>
</mat-card>
</div>
