<mat-toolbar>
    <mat-toolbar-row style="background-image: linear-gradient(-133deg, #246A92 0%, #245571 100%)"></mat-toolbar-row>
    <mat-toolbar-row style="background-image: linear-gradient(-133deg, #246A92 0%, #245571 100%)"></mat-toolbar-row>
</mat-toolbar>
<div class="row" style="font-family:'Lato'; margin-top:-75px!important; margin-left:20px; margin-right: 20px; align-items: center;" >
<mat-card class="about-page card card-body">
    <mat-card-content>
        <div style="font-family:'LatoBold'; font-size: 20px; margin-bottom: 5px;">Proteomics Data Commons FAQs</div>
        <div><ul>
                <li><b><a href="javascript:void(0);" class="faqs" (click)="scrollToElement(PDC_Goals)">What are the goals of PDC?</a></b></li>
                <li><b><a href="javascript:void(0);" class="faqs" (click)="scrollToElement(PDC_Launched)">Is PDC launched?</a></b></li>
                <li><b><a href="javascript:void(0);" class="faqs" (click)="scrollToElement(Access_Data)">Do I have to login to PDC to access data?</a></b></li>
                <li><b><a href="javascript:void(0);" class="faqs" (click)="scrollToElement(SubmitData)">How do I submit data to PDC? How do I register my program with PDC?</a></b></li>
                <li><b><a href="javascript:void(0);" class="faqs" (click)="scrollToElement(FindCorresponding)">How to find corresponding genomic and/or imaging data for the PDC proteomic data?</a></b></li>
                <li><b><a href="javascript:void(0);" class="faqs" (click)="scrollToElement(DifferencePDC_CPTAC)">What is the difference between PDC and CPTAC Data Portal?</a></b></li>
                <li><b><a href="javascript:void(0);" class="faqs" (click)="scrollToElement(Download_Data)">How do I download data?</a></b></li>
                <li><b><a href="javascript:void(0);" class="faqs" (click)="scrollToElement(Multiple_Files)">How do I download multiple files at once?</a></b></li>
                <li><b><a href="javascript:void(0);" class="faqs" (click)="scrollToElement(File_Error)">Why does the file download URLs throw an error or exception?</a></b></li>
                <li><b><a href="javascript:void(0);" class="faqs" (click)="scrollToElement(Details_Study)">Where do I get more details about a study?</a></b></li>
                <li><b><a href="javascript:void(0);" class="faqs" (click)="scrollToElement(Gene_Information)">How can I find gene/s of my interest?</a></b></li>
                <li><b><a href="javascript:void(0);" class="faqs" (click)="scrollToElement(Protein_Quantitaion)">How do I explore protein quantitation data for a study through heatmaps?</a></b></li>
                <li><b><a href="javascript:void(0);" class="faqs" (click)="scrollToElement(Search_PDC)">What can I search on PDC?</a></b></li>
                <li><b><a href="javascript:void(0);" class="faqs" (click)="scrollToElement(Search_Sequence)">Can I search for a particular peptide sequence within the datasets in PDC?</a></b></li>
                <li><b><a href="javascript:void(0);" class="faqs" (click)="scrollToElement(Files_Download)">What type of files are available for download from PDC?</a></b></li>
                <li><b><a href="javascript:void(0);" class="faqs" (click)="scrollToElement(Cite_PDC)">How do I cite the PDC?</a></b></li>
                <li><b><a href="javascript:void(0);" class="faqs" (click)="scrollToElement(Policies_PDC)">What are the data use and licensing policies of the PDC?</a></b></li>
                <li><b><a href="javascript:void(0);" class="faqs" (click)="scrollToElement(Member_Consortium)">Is PDC a member of the ProteomeXchange Consortium?</a></b></li>
                <li><b><a href="javascript:void(0);" class="faqs" (click)="scrollToElement(Report_Issue)">How do I report an issue or submit a comment about the process?</a></b></li>
                <li><b><a href="javascript:void(0);" class="faqs" (click)="scrollToElement(PDC_CDAP)">What is PDC Common Data Analysis pipeline? What is the PDC harmonization process?</a></b></li>
                <li><b><a href="javascript:void(0);" class="faqs" (click)="scrollToElement(Run_Resource)">Can I use PDC data from NCI Cloud Resource to run my own analyses?</a></b></li>
                <li><b><a href="javascript:void(0);" class="faqs" (click)="scrollToElement(Custom_Cohort)">Can I create a custom cohort to download?</a></b></li>
                <li><b><a href="javascript:void(0);" class="faqs" (click)="scrollToElement(Open_Access)">What data are not open-access?</a></b></li>
                <li><b><a href="javascript:void(0);" class="faqs" (click)="scrollToElement(Types_Data)">What types of data can I get using the API?</a></b></li>
                <li><b><a href="javascript:void(0);" class="faqs" (click)="scrollToElement(Ontologies_PDC)">What ontologies are used in annotating? Where can I learn about your metadata standards?</a></b></li>
        </ul></div>
        <div #PDC_Goals class="faq-question">What are the goals of PDC? </div>

        <p class="faq-response">The primary goal of Proteomic Data Commons (PDC) is to democratize access to cancer-related proteomic datasets and also aims to provide sustainable computational support to the cancer research community through interoperability with other nodes, such as Genomic Data Commons and Cloud Resources, in the NCI Cancer Research Data Commons (CRDC) ecosystem.</p>

        <div #PDC_Launched class="faq-question">Is PDC launched? </div>

			<p class="faq-response">NCI PDC is officially launched on Mar 23, 2020. Click <a href="https://proteomics.cancer.gov/news_and_announcements/proteomic-data-commons-now-live" target="_blank">here</a> to read the official announcement.</p>

      <div #Access_Data class="faq-question">Do I have to login to PDC to access data?</div>
      <p class="faq-response">No, there is no need to create an account or login to PDC to browse the portal or download the data. However login is required if you would like to submit data to PDC.</p>

      <div #SubmitData class="faq-question">How do I submit data to PDC? How do I register my program with PDC?</div>
      <p class="faq-response">Detailed instructions for submitting data into the PDC are available <a href="https://pdc.esacinc.com/pdc/submit-data" target="_blank">here</a>.</p>

      <div #FindCorresponding class="faq-question">How to find corresponding genomic and/or imaging data for the PDC proteomic data?</div>
      <p class="faq-response">If a study in PDC has corresponding genomic and/or imaging data available in other resources, you may find the mapping of the individual cases to the external resources in the <b>'Clinical'</b> tab on the browse page (see the image below). Either click on the thumbnail in the column 'Genomic or Imaging Data Resource' or you may also select the cases of interest and export the clinical manifest. The manifest file will have the resource name and identifier for each case.</p>
      <br><br><div style="text-align:center"><img style="width: 60%;margin-left: 9%;" src="assets/css/images/faq-5.png" alt="Select file manifect"></div>

      <div #DifferencePDC_CPTAC class="faq-question">What is the difference between PDC and CPTAC Data Portal?</div>
		<!-- @@@PDC-1711 add external link icon -->
		<p class="faq-response">The PDC will serve as a public data repository of mass spectrometry (MS) based proteomics data, and is maintained by the National Cancer Institute. PDC will host datasets from large consortiums such as Clinical Proteomic Tumor Analysis Consortium (CPTAC), International Cancer Proteogenome Consortium(ICPC) and Applied Proteogenomics OrganizationaL Learning and Outcomes (APOLLO) and also from independent research programs and grants.
            <br>The PDC Data Portal provides a platform for efficiently querying, visualizing, analyzing and downloading high quality, curated and harmonized proteomic datasets. All of the cancer cohort datasets from CPTAC data portal are already available on PDC.  In the near future the CPTAC data portal will retired and all of the data will be made available through PDC.
            <br>PDC represents a state of the art repository technology and facilitates multiomic integration through interoperability with other nodes, such as Genomic Data Commons and Cloud Resources, in the NCI Cancer Research Data Commons (CRDC) ecosystem.
            <br>The data from the Clinical Proteomic Tumor Analysis Consortium is also available from the <a href="https://cptac-data-portal.georgetown.edu/" target="_blank">CPTAC Data Portal</a><a matTooltip="Exit Disclaimer" href="https://www.cancer.gov/policies/linking" target="_">&nbsp;<i class="fas fa-external-link-alt"></i><span style="display: none">External Link</span></a>.
        </p>

        <div #Download_Data class="faq-question">How do I download data? </div>
        <p class="faq-response">Individual files can be downloaded by clicking the 'Download' button in the File table.
            <br>To download multiple files, identify and select the files of your interest by clicking the check box. You may select multiple files at once by clicking the checkbox in the header row of the table. After the files are selected, click on the 'Download Files' button. Currently there is a limit on the number and volume of files that can be downloaded through this approach.
            <br>If you want to download multiple files programmatically, see <a href="javascript:void(0);" class="faqs" (click)="scrollToElement(Multiple_Files)">'How do I download multiple files at once?'.</a></p>

        <div #Multiple_Files class="faq-question" id="Multiple_Files">How do I download multiple files at once?</div>
        <p class="faq-response">Once you identify the files of your interest, move to 'Files' tab and select one or all files and then click <strong><i>'Export File Manifest'</i></strong>.
            The manifest will contain a download URL each selected file in a separate row. <strong>The URLs will expire after 48 hours</strong>.
            You may revisit the PDC portal to generate a new file manifest.
            <br><br><div style="text-align:center"><img style="width: 60%;margin-left: 9%;" src="assets/css/images/faq-1.png" alt="Select file manifect"></div>
            <br>On most linux, mac and cygwin environments with wget installed, you may provide the <strong><i>manifest file</i></strong> as input to the following code snippet to initiate batch download.
            <br><br><div>
                    <pre style="background-color: black;color: white;width: 60%;float: center;margin-left: 25%;">
                    <span class="displayGrey">#!/bin/bash

                    #This is a sample bash script to batch download PDC data files in to your local computer
                    #This works on most linux machines
                    #The script uses only the native shell functionalities and wget
                    #Save this script as downloadPDCData.sh in your local directory and set executable permissions
                    #Provide downloaded PDC File manifest as a command line parameter
                    #Ex:  ./downloadPDCData.sh -f &lt;PDC File manifest&gt;
                    #Users may improvise using awk, sed and rewrite in perl, python, etc.</span>

                    helpFunction() {{ '{' }}
                        echo <span class="displayGreen">""</span>
                        echo <span class="displayGreen"> "Usage: $0 -f &lt;Name of the manifest file downloaded from PDC in CSV format&lt;"</span>
                        exit <span class="displayRed"> 1</span> <span class="displayGrey"># Exit script after printing help</span>
                    {{ '}' }}

                    <span class="displayOrange">while</span><span class="displayGreen"> getopts</span> ":f:" opt
                    do
                            case <span class="displayGreen">"$opt"</span> <span class="displayOrange">in</span>
                            f ) filename=<span class="displayGreen">"$OPTARG"</span> ;;
                            ? ) helpFunction ;; <span class="displayGrey"># Print helpFunction in case input file name is non-existent</span>
                            esac
                    done

                    <span class="displayGrey"># Print helpFunction in case input filename is empty</span>
                    <span class="displayOrange">if</span> [ -z <span class="displayGreen">"$filename"</span> ]
                    then
                        echo "Some or all of the parameters are empty";
                        helpFunction
                    fi

                    IFS=<span class="displayGreen">","</span> <span class="displayGrey">#split the input csv file based on comma separator and read into an array</span>

                    <span class="displayOrange">while</span> read -r -a array
                    do
                        <span class="displayOrange">for</span> index <span class="displayOrange">in</span> <span class="displayGreen">"${{ '{' }}!array[@]{{ '}' }}"</span>; do
                            <span class="displayGrey">#echo $index
                            #Extracting file URL from Column 11</span>
                            <span class="displayOrange">if</span> [[ <span class="displayGreen">"$index"</span> == <span class="displayRed">11</span>]; then
                                <span class="displayGrey">#echo "$index ${{ '{' }}array[$index]{{ '}' }}"</span>
                                url=<span class="displayGreen">"${{ '{' }}array[$index]{{ '}' }}"</span>
                                temp=<span class="displayGreen">"${{ '{' }}url%\&quot;{{ '}' }}&quot;</span> <span class="displayGrey">#will remove the suffix " (escaped with a backslash to prevent shell interpretation).</span>
                                temp=<span class="displayGreen">"${{ '{' }}temp#\&quot;{{ '}' }}&quot;</span> <span class="displayGrey">#will remove the prefix " (escaped with a backslash to prevent shell interpretation).</span>
                                url=$temp
                                echo <span class="displayGreen">"Downloading $url........."</span>
                                wget -c $url <span class="displayGrey"># -c allows resuming the failed or stopped downloads
                                                # -c also eliminates redownload if a file already exists (with same size) in the current directory.</span>
                            fi
                        done
                    done &gt; $filename
                    </pre>
            </div>
            <br>You may also use <strong>Download managers</strong> (special programs or browser extensions) that help manage large and multiple downloads.
            <br><br><div style="text-align:center">
                <img src="assets/css/images/faq-2.png" alt="An illustration of a download manager">
                <br><span style="font-size: 14px;font-weight: bold;">An illustration of a download manager</span>
            </div>
            <br>Some free download managers:
                <br><a href="https://www.freedownloadmanager.org/" target="_blank">https://www.freedownloadmanager.org/</a><a matTooltip="Exit Disclaimer" href="https://www.cancer.gov/policies/linking" target="_">&nbsp;<i class="fas fa-external-link-alt"></i><span style="display: none">External Link</span></a>
                <br><a href="https://jdownloader.org/" target="_blank">https://jdownloader.org/</a><a matTooltip="Exit Disclaimer" href="https://www.cancer.gov/policies/linking" target="_">&nbsp;<i class="fas fa-external-link-alt"></i><span style="display: none">External Link</span></a>
                <br><a href="http://xdman.sourceforge.net/ " target="_blank">http://xdman.sourceforge.net/</a><a matTooltip="Exit Disclaimer" href="https://www.cancer.gov/policies/linking" target="_">&nbsp;<i class="fas fa-external-link-alt"></i><span style="display: none">External Link</span></a>
            <br><i>Disclaimer: The third-party software links available on this website are provided "as is" without warranty of any kind, either expressed or implied and such software is to be used at your own risk.</i>
        </p>


        <div #File_Error class="faq-question">Why does the file download URLs throw an error or exception?</div>
        <p class="faq-response">PDC is hosted on AWS cloud and is under active development. To reduce egress costs from unintended downloads, <strong>the URLs will expire after 48 hours</strong>. You may revisit the PDC portal to generate a new file manifest.</p>

        <div #Details_Study class="faq-question">Where do I get more details about a study?</div>
        <p class="faq-response">Click on the name of the study in the browse page, that opens up a study summary page. The study summary page provides details about the objective of the study, protocol, experimental design, clinical data of the cases and samples used in the study.</p>

        <div #Gene_Information class="faq-question">How can I find gene/s of my interest?</div>
        <p class="faq-response">
            Genes can be searched with their gene symbols through the search box on PDC portal. Enter gene symbol or name of the gene (such as kinase) in the search box and click on the gene of your interest in the drop down. This will take you to a gene summary page.
            <br>It is also possible to search for multiple genes such as those involved in a pathway. Go to the Gene tab on the browse page and in the gene filter on the left hand side enter the list of gen symbols. You may also select from the prebuilt gene lists from several pathways important in cancer.
        </p>
        <div #Protein_Quantitaion class="faq-question">How do I explore protein quantitation data for a study through heatmaps?</div>
        <p class="faq-response">
            The PDC Common Data Analysis Pipeline generates a protein abundance matrix rolled up to gene level for each study. The data can be viewed as an interactive heatmap through Morpheus viewer on PDC.
            <br>Morhpes is a heat maps viewer from Broad Institute. It is versatile and has a lot of features to filter, cluster and save the data.
            <br>Here is a quick tutorial to find 500 most variables genes by variance and perform hierarchical clustering.
            <br>Go to study summary page and click on the heatmap thumbnail. It opens a new page with the quantitation preloaded into morpheus viewer.
           <br>More ways to explore the heatmap can be found here accessed here - <a href="https://software.broadinstitute.org/morpheus/tutorial.html" target="_blank">Morpheus - Tutorial</a><a matTooltip="Exit Disclaimer" href="https://www.cancer.gov/policies/linking" target="_">&nbsp;<i class="fas fa-external-link-alt"></i><span style="display: none">External Link</span></a>
        </p>

        <div #Search_PDC class="faq-question">What can I search on PDC?</div>
        <p class="faq-response">
          PDC can be searched for the following:
          <br>    - Biospecimens such as case, sample or aliquot using their original identifiers or PDC ids
          <br>    - Studies using their PDC id (e.g. PDC000220) or partial name (e.g. CPTAC, HNSCC)
          <br>    - Genes using their Gene symbol (e.g. BRCC3) or partial description (e.g. Kinase)
          <br>It is also possible to search for related data from external resources using the external identifiers. For example, you may search for a dbGaP study using its identifier (e.g., phs000892) to identify all related PDC studies.
        </p>


        <div #Search_Sequence class="faq-question">Can I search for a particular peptide sequence within the datasets in PDC?</div>
        <p class="faq-response">Peptides can be searched through PepQuery, a peptide-centric approach that focuses on only novel DNA or protein sequences of interest.
          <br>From the menu bar, go to Analysis -> PepQuery. Enter the peptide in the search box and select the dataset to search against. More details about PepQuery can be found here - <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6396417/" target="_blank">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6396417/</a>.</p>

        <div #Files_Download class="faq-question" id="Files_Download">What type of files are available for download from PDC? What are the outputs of PDC Common Data Analysis pipeline? What do they mean?</div>
          <div class="faq-response">
                PDC distributes several types of files, those submitted by original data submitters and those generated through the PDC common data analysis pipeline.<br><br>
                <table class="faqTable">
                        <tr><td colspan="2"><strong>Mass Spectrometry Data Formats</strong></td></tr>
                        <tr>
                            <td>RAW (Vendor) Format</td>
                            <td>Mass spectrometry data uploaded by the data submitters as RAW or vendor format files corresponding to the mass spectrometers used to acquire the spectra. </td>
                        </tr>
                        <tr>
                            <td>mzML Format</td>
                            <td>RAW format spectra in the <a href="http://www.psidev.info/" target="_blank">HUPO Proteome Standards Initiative (PSI)</a><a matTooltip="Exit Disclaimer" href="https://www.cancer.gov/policies/linking" target="_">&nbsp;<i class="fas fa-external-link-alt"></i><span style="display: none">External Link</span></a> compliant mzML format..</td>
                        </tr>
                        <tr><td colspan="2"><strong>Peptide-Spectrum Match (PSM) Data</strong></td></tr>
                        <tr>
                            <td>RAW PSM Format</td>
                            <td>The best peptide-spectrum matches (PSMs), from the  first-level analysis of the PDC CDAP, for each tandem-mass spectrum against the peptide sequences from a reference protein sequence database (Uniprot) in tsv format.</td>
                        </tr>
                        <tr>
                            <td>mzIdentML PSM Format</td>
                            <td>Raw PSMs in the PSI compliant mzIdentML format.</td>
                        </tr>
                        <tr><td colspan="2"><strong>Protein Assembly</strong></td></tr>
                        <tr>
                            <td></td>
                            <td>
                                Protein identification and quantitation reports generated from the PSM data through a conservative gene-based generalized parsimony analysis. Peptides are associated with genes, rather than protein identifiers, and genes with at least two unshared peptide identifications are inferred. The resulting gene list is estimated to have a false-discovery rate of at most 1%.
                                Several different output files are generated depending on the experiment type,
                                <br><strong>.summary.tsv</strong> - Protein identification summary report
                                <br><strong>.precursor_area.tsv</strong> - Label-free workflow protein quantitation report for relative quantitation by precursor peak area integration
                                <br><strong>.spectral_count.tsv</strong> - Label-free workflow protein quantitation report for relative quantitation by spectral counts
                                <br><strong>.itraq.tsv</strong> - iTRAQ workflow protein relative quantitation report
                                <br><strong>.tmt.tsv</strong> - TMT workflow protein relative quantitation report
                                <br><strong>.peptides.tsv</strong> - Identified peptide summary report
                                <br><strong>.phosphopeptide.tsv</strong> - Labelled workflow phosphopeptide relative quantitation report
                                <br><strong>.phosphosite.tsv</strong> - Labelled workflow phosphopeptide relative quantitation report
                                <br><strong>.glycopeptide.tsv</strong> - Labelled workflow N-linked glycopeptide relative quantitation report
                                <br><strong>.glycosite.tsv</strong> - Labelled workflow N-linked glycosite relative quantitation report
                            </td>
                        </tr>
                        <tr>
                            <td>QC reports</td>
                            <td>Quality control metrics computed by the CDAP, the report consists of summary statistics derived from all MS/MS spectra from the raw spectral datafiles.</td>
                        </tr>
                        <tr>
                            <td>Other metadata</td>
                            <td>
                                These are supplementary files provided by the original data submittres, these usually consists of,
                                <ul>
                                    <li>Descriptive protocols</li>
                                    <li>Clinical metadata</li>
                                    <li>Other useful information</li>
                                </ul>
                            </td>
                        </tr>
                </table>
        </div>

        <div #Cite_PDC class="faq-question">How do I cite the PDC?</div>
        <p class="faq-response">
                To cite the resource, cite PDC url - <a href="https://pdc.cancer.gov" target="_blank">https://pdc.cancer.gov</a>
            <br>A PDC study can be cited with its PDC study identifier or url to the study. Ex: PDC000220 or <a href="https://pdc.cancer.gov/pdc/study/PDC000220" target="_blank">https://pdc.cancer.gov/pdc/study/PDC000220</a>
        </p>

        <div #Policies_PDC class="faq-question">What are the data use and licensing policies of the PDC?</div>
        <p class="faq-response">
            PDC data submission and data use are governed by <a href="https://creativecommons.org/licenses/by/4.0/" target="_blank">Creative Commons CC-BY 0.4</a><a matTooltip="Exit Disclaimer" href="https://www.cancer.gov/policies/linking" target="_">&nbsp;<i class="fas fa-external-link-alt"></i><span style="display: none">External Link</span></a> licensing terms.
            <br>Under CC-BY terms, that user is free to,
            <br>&nbsp;&nbsp;<sup><i class="fa fa-circle"></i></sup>&nbsp; Share — copy and redistribute the material in any medium or format
            <br>&nbsp;&nbsp;<sup><i class="fa fa-circle"></i></sup>&nbsp; Adapt — remix, transform, and build upon the material for any purpose, even commercially.
            <br>The licensor cannot revoke these freedoms as long as you follow the license terms.
            <br>Under the following terms:
            <br>&nbsp;&nbsp;<sup><i class="fa fa-circle"></i></sup>&nbsp; Attribution — You must give <a href="https://creativecommons.org/licenses/by/4.0/#" target="_blank">appropriate credit</a><a matTooltip="Exit Disclaimer" href="https://www.cancer.gov/policies/linking" target="_">&nbsp;<i class="fas fa-external-link-alt"></i><span style="display: none">External Link</span></a>, provide a link to the license, and <a href="https://creativecommons.org/licenses/by/4.0/#" target="_blank">indicate if changes were made</a><a matTooltip="Exit Disclaimer" href="https://www.cancer.gov/policies/linking" target="_">&nbsp;<i class="fas fa-external-link-alt"></i><span style="display: none">External Link</span></a>. You may do so in any reasonable manner, but not in any way that suggests the licensor endorses you or your use.
            <br>&nbsp;&nbsp;<sup><i class="fa fa-circle"></i></sup>&nbsp; No additional restrictions — You may not apply legal terms or <a href="https://creativecommons.org/licenses/by/4.0/#" target="_blank">technological measures</a><a matTooltip="Exit Disclaimer" href="https://www.cancer.gov/policies/linking" target="_">&nbsp;<i class="fas fa-external-link-alt"></i><span style="display: none">External Link</span></a> that legally restrict others from doing anything the license permits.
            <br><strong>Note:</strong>
            <br>PDC may impose an embargo on certain datasets at the request of data submitters. In PDC, an <strong>embargo</strong> is the release of data on the PDC portal with the condition that users who access these datasets from the PDC portal cannot published or disseminated before a certain <strong>date</strong>.
            The embargo date is displayed on the study summary page of each dataset as applicable.
        </p>

        <div #Member_Consortium class="faq-question">Is PDC a member of the ProteomeXchange Consortium?</div>
        <p class="faq-response">PDC is currently an observer member of the ProteomeXchange Consortium.</p>

        <div #Report_Issue class="faq-question">How do I report an issue or submit a comment about the process?</div>
        <p class="faq-response">You can send us your feedback or comments on any issues you experience at <a href="mailto:nci.pdc.help@esacinc.com?Subject=PDC Help" target="_top">nci.pdc.help@esacinc.com.</a></p>

        <div #PDC_CDAP class="faq-question">What is PDC Common Data Analysis pipeline? What is the PDC harmonization process? What tools have been used to produce the processed data?</div>
        <p class="faq-response">Learn more about the PDC harmonization process <a href="https://pdc.esacinc.com/data-dictionary/harmonization.html" target="_blank">here</a>.</p>

        <div #Run_Resource class="faq-question">Can I use PDC data from NCI Cloud Resource to run my own analyses?</div>
        <p class="faq-response">PDC data is located on Amazon Web Services S3 cloud buckets. All of the PDC data and metadata is accessible through APIs. There is an ongoing effort within the Cancer Research Data Commons for the Cloud Resources to run analysis pipelines on PDC data.
            <br>A tutorial on navigating the database is available from <a href="http://www.cancergenomicscloud.org/" target="_blank">Seven Bridges Cancer Genomic Cloud (CGC)</a><a matTooltip="Exit Disclaimer" href="https://www.cancer.gov/policies/linking" target="_">&nbsp;<i class="fas fa-external-link-alt"></i><span style="display: none">External Link</span></a>, one of NCI’s Cloud Resources. Using the Seven Bridges “Data Cruncher,” and a specially designed PDC metadata file (PDC_metadata.json) that’s available in the public reference resources, users can extract data from the PDC. Other NCI Cloud Resources include the Broad Institute’s FireCloud and the Institute for Systems Biology’s Cancer Genomics Cloud.
        </p>

        <div #Custom_Cohort class="faq-question">Can I create a custom cohort to download?</div>
        <p class="faq-response">PDC portal allows users to build cohorts by applying various clinical, biospecimen, experimental and file features as filters and export the selections as manifest files.</p>

        <div #Open_Access class="faq-question">What data are not open-access?</div>
        <p class="faq-response">Currently all of the data in PDC is open access.
            <br>In the near future, PDC is expected to host data submitters provided patient sample specific protein sequence databases. These databases are generated using the genomic and transcriptomic information from the patient sample and helps in identification of novel proteins, resulting from single nucleotide variants, splice variants and fusion genes. These files are currently designated as controlled access databases and would need dbGaP authorization for access.
        </p>
        <div #Types_Data class="faq-question">What types of data can I get using the API?</div>
        <p class="faq-response">All of the PDC data and metadata is accessible through APIs. Learn more <a href="https://pdc.esacinc.com/data-dictionary/apidocumentation.html" target="_blank">here</a>.</p>

        <div #Ontologies_PDC class="faq-question">What ontologies are used in annotating? Where can I learn about your metadata standards?</div>
        <p class="faq-response">PDC uses <a href="https://wiki.nci.nih.gov/pages/viewpage.action?pageId=309494141&sortBy=date&sortOrder=ascending" target="_blank">caDSR</a> and <a href="https://ncithesaurus.nci.nih.gov/ncitbrowser/" target="_blank">NCIt</a> terminologies for biospecimen and clinical metadata and <a href="http://www.psidev.info/groups/controlled-vocabularies" target="_blank">PSI</a><a matTooltip="Exit Disclaimer" href="https://www.cancer.gov/policies/linking" target="_">&nbsp;<i class="fas fa-external-link-alt"></i><span style="display: none">External Link</span></a> ontologies for Proteomic metadata. <a href="https://pdc.esacinc.com/data-dictionary/dictionary.html" target="_blank">Learn</a> more about PDC data dictionary and data model.</p>
    </mat-card-content>
</mat-card>
</div>
<mat-card>
    <mat-card-title>Contact Us</mat-card-title>
    <mat-card-content>
        Feedback? Join our Google discussion group: <a href="https://groups.google.com/a/esacinc.com/forum/#!forum/pdc-mvp-feedback" target="_blank">PDC Feedback Group</a><a matTooltip="Exit Disclaimer" href="https://www.cancer.gov/policies/linking" target="_">&nbsp;<i class="fas fa-external-link-alt"></i><span style="display: none">External Link</span></a>
        <br>
        Contact us: <a href="mailto:nci.pdc.help@esacinc.com?Subject=PDC Help" target="_top">Email</a>
    </mat-card-content>
</mat-card>
