<mat-toolbar>
  <mat-toolbar-row style="background-image: linear-gradient(-133deg, #246A92 0%, #245571 100%)"></mat-toolbar-row>
  <mat-toolbar-row style="background-image: linear-gradient(-133deg, #246A92 0%, #245571 100%)"></mat-toolbar-row>
</mat-toolbar>
<div class="row" id="exploreQuantitationDiv" style="font-family:'Lato'; margin-top:-75px!important; margin-left:20px; margin-right: 20px; align-items: center;" >
<mat-card class="card card-body">
  <mat-card-content style="margin-bottom: 13%">
    <div #Top style="font-family:'LatoBold'; font-size: 20px; margin-bottom: 5px;">Biological interpretation of PDC protein expression data through heatmaps</div><br>
    <div>
      <p>The PDC processes the submitted raw mass spectrometry data through a common data analysis pipeline (CDAP) to produce derived analysis results which can be used to study identification of proteins and post-translational modifications (PTMs). On the PDC portal, these quantitative results can be visualized as a heatmap for downstream analyses of differential expression between samples at the protein- or PTM site-level.</p>
      <br><br>
      <div class="row">
				 <div class="col-lg-12 col-md-12 col-sm-12 col-xs-12" style="margin-left: -1.3%;">
						<div class="col-lg-6 col-md-6 col-sm-12 col-xs-12" style="margin-top: -1.5%;">
                <h3>Software and Access</h3>
                The PDC includes an implementation of the client-side application, Morpheus, developed at the Broad Institute. The Mopheus <a href="https://software.broadinstitute.org/morpheus/" target="_blank">website</a> offers <a href="https://software.broadinstitute.org/morpheus/documentation.html" target="_blank">documentation</a>, a <a href="https://software.broadinstitute.org/morpheus/tutorial.html" target="_blank">tutorial</a>, notes on <a href="https://software.broadinstitute.org/morpheus/configuration.html" target="_blank">configuration</a>, and other resources.  Morpheus on the PDC can be accessed primarily through the heatmap image links (see image at right) displayed on the study page (e.g., <a href="https://pdc.cancer.gov/pdc/study/PDC000120" target="_blank">PDC000120</a>), where each heatmap automatically loads the quantitative data for the selected study into the application.
            </div>
            <div class="col-lg-1 col-md-1 col-sm-0 col-xs-0"></div>
            <div class="col-lg-5 col-md-5 col-sm-12 col-xs-12">
                <img id="exploreQuantitation1" src="assets/exploreQuantitation1.png" alt="Software and Access" style= "height: 300px">
						</div>
          </div>
      </div>
      <br><br>
      <div class="row">
          <div class="col-lg-12 col-md-12 col-sm-12 col-xs-12" style="margin-left: -1.3%;">
             <div class="col-lg-6 col-md-6 col-sm-12 col-xs-12" style="margin-top: -1.5%;">
                <h3>Hierarchical clustering</h3>
                The main purpose of heatmap visualization, in the context of the PDC, is to examine <a href="https://en.wikipedia.org/wiki/Hierarchical_clustering" target="_blank">clustering</a> of samples according to protein abundances.  In a typical heatmap visualization, rows correspond to genes (or proteins) and columns correspond to samples (or aliquots.)  Additional row (e.g., genes accessions) and column (e.g., clinical attributes) annotations can be embedded in the input file read by Morpheus.
                <br>When a study heatmap is loaded, the data have not been clustered by either row or column.  Clustering can be performed according to the method of your choice by accessing the Tools menu.  You will know your data have been clustered if above the column annotations is displayed a dendogram, linking the samples.  In addition, the ordering of the columns will have changed according to the clustering calculations.  Morpheus allows for a large variety of data analysis methods.  If you are unfamiliar with the statistical methods available in the application, you may wish to visit the Morpheus site or consult with bioinformatics colleagues.             
             </div>
             <div class="col-lg-1 col-md-1 col-sm-0 col-xs-0"></div>
             <div class="col-lg-5 col-md-5 col-sm-12 col-xs-12">
                 <img id="exploreQuantitation2" src="assets/hierarchialClustering.png" alt="Hierarchical clustering" style= "height: 300px">
             </div>
           </div>
      </div>
      <br><br>
      <div class="row">
          <div class="col-lg-12 col-md-12 col-sm-12 col-xs-12" style="margin-left: -1.3%;">
             <div class="col-lg-6 col-md-6 col-sm-12 col-xs-12" style="margin-top: -1.5%;">
                <h3>Data</h3>
                Analysis results from the CPTAC CDAP (Common Data Analysis Pipeline) are the source data for heatmap visualizations on the PDC.  These data are merged with various row and column annotations to generate a .gct file when the study is loaded into the PDC.  For proteome studies, both log2 ratio and unshared_log2_ratio data are available for visualization.  Unshared_log2_ratio data exclude data from shared peptides.  Each cell represents the normalized log2 ratio from an iTRAQ or TMT experiment, where the numerator is the value for that gene/protein in that aliquot and the denominator is the expression level of whatever control channel was used in the study.  Control channels are typically pooled samples from many tumor samples.
                <br>In addition to downloading publication-quality images from Morpheus, you may also export the data in .gct format for use with statistical analysis tools (e.g., in R or Python.)  Exports are available from the File menu.  We encourage you to try the available tools available from the Morpheus menu, including those under View->Options to help you identify potential correlations between annotation variables and the protein expression data.
             </div>
             <div class="col-lg-1 col-md-1 col-sm-0 col-xs-0"></div>
             <div class="col-lg-5 col-md-5 col-sm-12 col-xs-12">
                 <img id="exploreQuantitation3" src="assets/exploreQuantitation3.png" alt="Data" style= "height: 300px">
             </div>
           </div>
      </div>
    </div>
  </mat-card-content>
</mat-card>
</div>
